src/zoom.js

```diff
@@ -282,21 +294,64 @@ export default function() {
     interrupt(this);
     g.start();
 
+    startX = x0;
+    startY = y0;
+
+   cancelMomentumTracking();
```

Comment:
Minor formatting inconsistency

```suggestion
    cancelMomentumTracking();
```